test/test_transforms.py

```diff
@@ -28,7 +28,227 @@
 
 GRACE_HOPPER = get_file_path_2(
     os.path.dirname(os.path.abspath(__file__)), 'assets', 'encode_jpeg', 'grace_hopper_517x606.jpg')
+	
 
+def test_random_affine():
+    with pytest.raises(ValueError):
+        transforms.RandomAffine(-0.7)
```

Comment:
We'll need 1 `raises` block per line here, because we need to make sure that each individual line raises a ValueError. That's not your fault: the original code was wrong I think